connector/dialect.tdd

```diff
@@ -1196,9 +1196,16 @@
       <formula>EXTRACT(%1 FROM %2)</formula>
       <!-- MongoDB week number starts at 0 and not 1 -->
       <formula part='week'>(1 + EXTRACT(%1 FROM %2))</formula>
+      <formula part='weekday'>EXTRACT(%1 FROM DATEADD(DAY, 1, %2))</formula>
```

Comment:
I noticed that 'weekday' was not implemented, but 'iso_weekday' was close, although it started on Monday. So I used the similar idea with the 'week' formula and used dateadd to shift the day one over.

Reply:
clever